### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by jvyden**
This is just from my understanding of how Kettu works, feel free to change anything.

(also add hacktoberfest to project tags so i can retrieve my clout)

### CODE REVIEW

File: Kettu/LoggerBase.cs [hunk 1]
```diff
@@ -2,6 +2,9 @@
 using System.Collections.Generic;
 
 namespace Kettu {
+    /// <summary>
+    /// A method of logging LoggerLines to a file.
+    /// </summary>
```
Comment (Beyley) [inline Kettu/LoggerBase.cs:7]: what

**Review by Beyley — CHANGES_REQUESTED** [review]

**Comment by Beyley** [comment]
One more thing: you forgot to document LoggerBase.Dispose

File: Kettu/LoggerBase.cs (hunk 1 above)
Comment (jvyden) [reply Kettu/LoggerBase.cs]: oops

**Comment by jvyden** [comment]
The button should be near the top-right of the front page.

**Review by Beyley — APPROVED** [review]

**Comment by Beyley** [comment]
tag added too btw

**Pull request merged**